connectivity/mbedtls/platform/inc/platform_mbed.h

```diff
@@ -50,6 +50,8 @@
 
 #define MBEDTLS_ENTROPY_HARDWARE_ALT
 
+#define MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT
```

Comment:
This is enabled in "mbedtls: Initialize PSA Crypto if available", even for non-PSA configurations, but no implementation is provided for non-PSA (and non-cryptocell) configurations.

Reply:
This is enabled for PSA only - see lines 24 and 55.

Reply:
OK, I missed this was in a PSA only section.